web/.htaccess

```diff
@@ -0,0 +1,4 @@
+RewriteEngine ON
```

Comment:
@dariama3 Please, fix domains in `docker-compose.yml` and other files as required. See Lesson 1, slide #33.